app/services/scrape.py

```diff
@@ -218,3 +235,62 @@ def get_frequency(self, user_id: str) -> Tuple[TimeSeries, TimeSeries]:
             value=hour_of_day.tolist(),
         )
         return dow_resp, hod_resp
+
+    def get_ml_user(self, username: str):
```

Comment:
ping @dirtyg2120, who create class `ML`

Reply:
Then `TwitterScraper` should not call `ML` because `ML` also need to call `TwitterScraper` to get input for model, which will create a circular import

Reply:
hmm, `ML` can call directly from db. Because if call from `TwitterScraper`, maybe new data will be scraped instead the one we save from report.

Reply:
put both of it into 1 file is fine, but will it make the file more complicated to read?

Reply:
I think I can move `get_ml_*` methods to `ML` class. About circular import issue, it is best solved if `TwitterScraper` don't call `ML` score at all, and also get score should not be responsibility of `TwitterScraper` class.

Reply:
agree :+1:

Reply:
I have tried to separate `TwitterScraper` from `ML` in previous PR #25 but you guys don't want it

Reply:
> TwitterScraper` don't call `ML` score at all, and also get score should not be responsibility of `TwitterScraper` class.

let see wat @produdez think. If i didn't want it previous PR, maybe it just "sai lầm quá khứ" :V

Reply:
Yeah facade class sound like a good idea